src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderInput.tsx

```diff
@@ -270,19 +275,19 @@ export const LimitOrderInput = ({
         sellAmountCryptoPrecision={sellAmountCryptoPrecision}
         sellAmountUserCurrency={sellAmountUserCurrency}
         sellAsset={sellAsset}
-        sellAssetAccountId={sellAssetAccountId}
+        sellAssetAccountId={sellAccountId}
```

Comment:
same as previously re: keeping the current terminology or making it consistent in props (preference for the former unless there's a rationale for the renaming)